### PR TITLE
New Package: zita-convolver-3.1.0

### PR DESCRIPTION
**Pull request by silvernode**
There was a warning message. Not really sure why.

WARNING: zita-convolver-3.1.0_1: libzita-convolver.so.3 not found in common/shlibs!

### CODE REVIEW

File: srcpkgs/zita-convolver/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'zita-convolver'
+pkgname=zita-convolver
+version=3.1.0
+revision=1
+build_style=gnu-makefile
+make_install_args="LIBDIR=lib"
+hostmakedepends="pkg-config"
+makedepends="fftw-devel"
+depends="fftw"
```
Comment (xtraeme) [inline srcpkgs/zita-convolver/template]: This is wrong...

File: srcpkgs/zita-convolver-devel/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'zita-convolver'
```
Comment (xtraeme) [inline srcpkgs/zita-convolver-devel/template]: zita-devel must be a symlink to srcpkgs/zita, not a whole file.

**Comment by silvernode** [comment]
TIL devel templates must be a symlink not a whole file.

**Comment by silvernode** [comment]
I will get to this as soon as I get a break from working. Just wanted to pop in and let everyone know I have been monitoring the comments.

**Pull request closed without merging**